ScruffyPigeon/developer-test
# Issue3

**Change Description**
File - package.json
Desc 
- Automatic newline addition

File - public/resources/sass/base/_typography.scss
Desc
- Changed font colour of 'small' html elements found in 'h4' html elements to use $gray-base for ease of visibility

File - public/resources/sass/components/_episode.scss
Desc
- Removed line-height styling for 'p' html elements that appear inside html elements with the class 'episode'
- Decreased font-size for 'p' html elements that appear inside html elements with the class 'episode'
- Set margin to 0 for 'p' html elements that appear inside html elements with the class 'episode'
- Added new styles for media-heading class, with settings for font-weight, font-size, and line-height

File - public/resources/sass/layout/_global.scss
Desc
- Added new styles for 'row' class with a setting for margin-bottom

---

**Testing**
Steps
- Open browser
- Navigate to index page
- Compare the index page layout and styles with the design.png image provided
- Verify similarities

---

**Deployment steps**
- Merge branch 'ScruffyPigeon/developer-test` - 'ISSUE3` into 'ScruffyPigeon/developer-test` - `master`
- Compile assets: `./node_modules/.bin/gulp`
